src/app/_components/Table/TooltipHeader.tsx

```diff
@@ -10,12 +12,15 @@ type TableHeaderWithTooltipProps = {
   description: TooltipProps['description']
 }
 
-const TOUCH_TARGET = {
-  iconSize: 20,
-  iconWrapperPadding: 'p-1.5',
-  class: 'p-2',
-  offsetClass: '-m-2',
-} as const
+const TOUCH_TARGET: TouchTarget = {
+  touchAreaPadding: 'p-2',
+  touchAreaOffset: '-m-2',
+}
+
+const TOUCH_TARGET_ICON: TouchTarget = {
+  visibleElementSize: 20,
+  touchAreaPadding: 'p-1.5',
+}
```

Comment:
What about something like:

```suggestion
const TOUCH_TARGET_ICON: TouchTarget & { visibleElementSize: number } = {
  visibleElementSize: 20,
  touchAreaPadding: 'p-1.5',
};
```

Reply:
love it!